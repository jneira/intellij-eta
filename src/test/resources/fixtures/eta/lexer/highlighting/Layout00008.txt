ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00008')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITblockComment ('-- | Calculate the normal form CRC using a table\n')
ITvarid ('crcNormal')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
IToparen ('(')
ITconid ('Bits')
WHITE_SPACE (' ')
ITvarid ('a')
ITcparen (')')
WHITE_SPACE ('\n          ')
ITdarrow ('=>')
WHITE_SPACE (' ')
ITconid ('Pull1')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Data')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Pull1')
WHITE_SPACE (' ')
ITconid ('Word8')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Data')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE ('\n')
ITvarid ('crcNormal')
WHITE_SPACE (' ')
ITvarid ('table')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE (' ')
ITvarid ('xs')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('fromZero')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('fold')
WHITE_SPACE (' ')
ITvarid ('step')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE (' ')
ITvarid ('xs')
WHITE_SPACE ('\n  ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvarid ('sz')
WHITE_SPACE ('         ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('bitSize')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE ('\n    ')
ITvarid ('step')
WHITE_SPACE (' ')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('table')
WHITE_SPACE (' ')
ITvarsym ('!')
WHITE_SPACE (' ')
IToparen ('(')
ITconid ('Z')
WHITE_SPACE (' ')
ITconsym (':.')
WHITE_SPACE (' ')
ITvarid ('i2n')
WHITE_SPACE (' ')
IToparen ('(')
IToparen ('(')
ITvarid ('i2n')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarsym ('.>>.')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('sz')
WHITE_SPACE (' ')
ITvarsym ('-')
WHITE_SPACE (' ')
ITinteger ('8')
ITcparen (')')
ITcparen (')')
WHITE_SPACE (' ')
ITvarsym ('.&.')
WHITE_SPACE (' ')
ITinteger ('0xFF')
ITcparen (')')
WHITE_SPACE (' ')
ITvarid ('`xor`')
WHITE_SPACE (' ')
ITvarid ('a')
ITcparen (')')
ITcparen (')')
ITcparen (')')
WHITE_SPACE (' ')
ITvarid ('`xor`')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarsym ('.<<.')
WHITE_SPACE (' ')
ITinteger ('8')
ITcparen (')')
WHITE_SPACE ('\n')
