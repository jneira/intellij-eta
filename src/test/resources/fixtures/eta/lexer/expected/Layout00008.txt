ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00008')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITlineComment ('-- | Calculate the normal form CRC using a table')
WHITE_SPACE ('\n')
ITvocurly ('')
ITvarid ('crcNormal')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
IToparen ('(')
ITconid ('Bits')
WHITE_SPACE (' ')
ITvarid ('a')
ITcparen (')')
WHITE_SPACE ('\n          ')
ITdarrow ('=>')
WHITE_SPACE (' ')
ITconid ('Pull1')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Data')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Pull1')
WHITE_SPACE (' ')
ITconid ('Word8')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Data')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE ('\n')
ITsemi ('')
ITvarid ('crcNormal')
WHITE_SPACE (' ')
ITvarid ('table')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE (' ')
ITvarid ('xs')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('fromZero')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('fold')
WHITE_SPACE (' ')
ITvarid ('step')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE (' ')
ITvarid ('xs')
WHITE_SPACE ('\n  ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvocurly ('')
ITvarid ('sz')
WHITE_SPACE ('         ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('bitSize')
WHITE_SPACE (' ')
ITvarid ('initial')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarid ('step')
WHITE_SPACE (' ')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('table')
WHITE_SPACE (' ')
ITbang ('!')
WHITE_SPACE (' ')
IToparen ('(')
ITconid ('Z')
WHITE_SPACE (' ')
ITconsym (':.')
WHITE_SPACE (' ')
ITvarid ('i2n')
WHITE_SPACE (' ')
IToparen ('(')
IToparen ('(')
ITvarid ('i2n')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarsym ('.>>.')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('sz')
WHITE_SPACE (' ')
ITminus ('-')
WHITE_SPACE (' ')
ITinteger ('8')
ITcparen (')')
ITcparen (')')
WHITE_SPACE (' ')
ITvarsym ('.&.')
WHITE_SPACE (' ')
ITinteger ('0xFF')
ITcparen (')')
WHITE_SPACE (' ')
ITbackquote ('`')
ITvarid ('xor')
ITbackquote ('`')
WHITE_SPACE (' ')
ITvarid ('a')
ITcparen (')')
ITcparen (')')
ITcparen (')')
WHITE_SPACE (' ')
ITbackquote ('`')
ITvarid ('xor')
ITbackquote ('`')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('crc')
WHITE_SPACE (' ')
ITvarsym ('.<<.')
WHITE_SPACE (' ')
ITinteger ('8')
ITcparen (')')
WHITE_SPACE ('\n')
ITvccurly ('')
ITsemi ('')